src/CommunityToolkit/Xamarin.CommunityToolkit/Views/CameraView/iOS/FormsCameraView.ios.cs

```diff
@@ -239,6 +239,21 @@ public async Task TakePhoto()
 			}
 		}
 
+		AVCaptureFlashMode GetFlashMode()
+		{
+			switch (flashMode)
+			{
+				default:
+				case CameraFlashMode.Off:
+					return AVCaptureFlashMode.Off;
+				case CameraFlashMode.On:
+				case CameraFlashMode.Torch:
+					return AVCaptureFlashMode.On;
+				case CameraFlashMode.Auto:
+					return AVCaptureFlashMode.Auto;
```

Comment:
Just changing the `default` order to match with the rest of our code
```suggestion
				case CameraFlashMode.On:
				case CameraFlashMode.Torch:
					return AVCaptureFlashMode.On;
				case CameraFlashMode.Auto:
					return AVCaptureFlashMode.Auto;
				case CameraFlashMode.Off:
				default:
					return AVCaptureFlashMode.Off;
```

Reply:
@pictos done 👍